### PR TITLE
Fix upgrade tests for GKE Regional Clusters

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: test/e2e/framework/nodes_util.go [hunk 1]
```diff
@@ -119,17 +119,34 @@ func masterUpgradeGCE(rawV string, enableKubeProxyDaemonSet bool) error {
...
 		"clusters",
 		fmt.Sprintf("--project=%s", TestContext.CloudConfig.ProjectID),
-		fmt.Sprintf("--zone=%s", TestContext.CloudConfig.Zone),
+		locationParamGKE(),
 		"upgrade",
 		TestContext.CloudConfig.Cluster,
 		"--master",
 		fmt.Sprintf("--cluster-version=%s", v),
 		"--quiet")
+	_, _, err := RunCmd("gcloud", gcloudContainerCommandGroup(), args...)
```
Comment (mborsz) [inline test/e2e/framework/nodes_util.go]: Does it work for zonal case?

When I'm calling "gcloud '' container clusters list" I'm getting: ERROR: (gcloud) Invalid choice: ''.

Comment (wojtek-t) [reply test/e2e/framework/nodes_util.go]: It should already be fixed now :)

**Comment by krzyzacy** [comment]
is it going to affect non-regional upgrade tests?

**Comment by wojtek-t** [comment]
> is it going to affect non-regional upgrade tests?

It shouldn't. If it does, then it's an unintentional bug.

**Comment by krzyzacy** [comment]
@wojtek-t you probably also need to cherrypick this back to older releases, since we do 1.7-1.8, and 1.8-1.9 upgrades for regional cluster

**Pull request merged**